galleries/examples/showcase/pan_zoom_overlap.py

```diff
@@ -0,0 +1,70 @@
+"""
+===================================
+Pan/zoom events of overlapping axes
+===================================
+
+Example to illustrate how pan/zoom events of overlapping axes are treated.
+
+
+The default is the following:
+
+- Axes with a visible patch capture pan/zoom events
+- Axes with an invisible patch forward pan/zoom events to axes below
+- Shared axes always trigger with their parent axes
+  (irrespective of the patch visibility)
+
+
+``ax.set_forward_navigation_events(val)`` can be used to override the
+default behaviour:
+
+- ``True``:  Forward navigation events to axes below.
+- ``False``: Execute navigation events only on this axes.
+- ``"auto"``: Use the default behaviour
+  (``True`` for axes with an invisible patch and ``False`` otherwise).
```

Comment:
be explicit about what "invisible" means here (`obj.set_visible(False)`)

Reply:
added a note on what is considered a "visible patch"